### PR TITLE
Zigbee lock fixes

### PR DESCRIPTION
**Pull request by varzac**

### CODE REVIEW

**Comment by varzac** [comment]
I forgot to update some of the tests to reflect the `state_change = true` for rename events.  I will do so.

File: drivers/SmartThings/zigbee-lock/src/yale-fingerprint-lock/init.lua [hunk 1]
```diff
@@ -17,7 +17,7 @@ local capabilities = require "st.capabilities"
 local LockCluster = clusters.DoorLock
 local LockCodes = capabilities.lockCodes
 
-local lock_constants = (require "lock_constants")
+local lock_utils = (require "lock_utils")
```
Comment (greens) [inline drivers/SmartThings/zigbee-lock/src/yale-fingerprint-lock/init.lua:20]: either this was unused or some stuff needs to be renamed

File: drivers/SmartThings/zwave-lock/src/schlage-lock/init.lua [hunk 2]
```diff
@@ -54,26 +54,35 @@ local function reload_all_codes(self, device, cmd)
...
+      capability = capabilities.lockCodes.ID,
+      command = capabilities.lockCodes.commands.nameSlot.NAME,
+      args = cmd.args,
+      positional_args = cmd.positional_args
+    })
   else
-    send_set_user_code()
+    -- copied from defaults with additional check for Schlage's configuration
+    if (cmd.args.codeName ~= nil and cmd.args.codeName ~= "") then
+      if (device:get_field(constants.CODE_STATE) == nil) then device:set_field(constants.CODE_STATE, {}) end
```
Comment (greens) [inline drivers/SmartThings/zwave-lock/src/schlage-lock/init.lua:67]: this file also needs the `lock_constants` -> `lock_utils` change

Comment (varzac) [reply drivers/SmartThings/zwave-lock/src/schlage-lock/init.lua:67]: This is in the zwave lock which I haven't touched yet.  The only reason changes are showing up here is that I based this off your PR, so it included the rename fix.

Comment (varzac) [reply drivers/SmartThings/zwave-lock/src/schlage-lock/init.lua:66]: I should probably try to fix the commits though so this isn't included in a commit for "Zigbee lock code fixes"

**Review by greens — APPROVED** [review]

**Review by greens — APPROVED** [review]

**Pull request merged**